6-async/01-callbacks/article.md

```diff
@@ -213,32 +213,33 @@ loadScript('1.js', function(error, script) {
 });
 ```
 
-In the code above:
-1. We load `1.js`, then if there's no error.
-2. We load `2.js`, then if there's no error.
-3. We load `3.js`, then if there's no error -- do something else `(*)`.
+En el código de arriba:
+
+1. Cargamos `1.js`, entonces, si no hay error.
+2. Cargamos `2.js`, entonces, si no hay error.
+3. Cargamos `3.js`, entonces, si no hay error -- hacemos algo más `(*)`.
 
-As calls become more nested, the code becomes deeper and increasingly more difficult to manage, especially if we have a real code instead of `...`, that may include more loops, conditional statements and so on.
+Mientras llamadas se hagan más anidades, el código se vuelve más profundo y incrementalmente más difícil de manejar, especialmente si tenemos código real en lugar de `...`, que puede incluír más bucles, declaraciones de condiciones y más.
 
-That's sometimes called "callback hell" or "pyramid of doom."
+Esto es a lo que algunas veces llamamos "infierno de callback" o "pirámide de la catástrofe."
```

Comment:
Se podría agregar los términos en ingles para mejore referencia.

"callback hell" 
 "pyramid of doom.